src/applications/pact-act/containers/questions/agent-orange/AgentOrange-2-2-1-B.jsx

```diff
@@ -74,11 +74,11 @@ const mapDispatchToProps = {
 
 Orange221B.propTypes = {
   formResponses: PropTypes.object.isRequired,
-  setOrange221B: PropTypes.func.isRequired,
-  viewedIntroPage: PropTypes.bool.isRequired,
   router: PropTypes.shape({
     push: PropTypes.func,
-  }),
+  }).isRequired,
```

Comment:
We had some inconsistency in `router` prop type declarations. This change is in the name of consistency.